Unit tests fail on Node 8

``` 
57 passing (6s)
12 failing

  1) Thorn Fixtures creation should create a fixture:
     Uncaught AssertionError: expected undefined to equal 'Fixtures'
      at Interceptor.body (tests/index.js:130:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  2) Thorn Fixtures creation should create a fixture using options.module:
     Uncaught AssertionError: expected undefined to equal 'Fixtures'
      at Interceptor.body (tests/index.js:167:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  3) Thorn Fixtures creation should create multiple fixtures:
     Uncaught AssertionError: expected undefined to equal 'Fixtures'
      at Interceptor.body (tests/index.js:259:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  4) Thorn Fixtures linking should create fixtures and link them:
     Uncaught AssertionError: expected undefined to equal 'Fixtures'
      at Interceptor.body (tests/index.js:452:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  5) Thorn Fixtures linking should retry fixture creation and linking on 401's:
     Uncaught AssertionError: expected undefined to equal 'Fixtures'
      at Interceptor.body (tests/index.js:495:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  6) Thorn Fixtures linking with pre-existing records should link records:
     Uncaught AssertionError: expected undefined to equal 'Fixtures'
      at Interceptor.body (tests/index.js:536:68)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  7) Thorn Agent request methods should send GET request:
     Uncaught AssertionError: expected undefined to equal 'Agent-foo'
      at Interceptor.body (tests/index.js:882:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:566:14)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  8) Thorn Agent request methods should send POST request:
     Uncaught AssertionError: expected undefined to equal 'Agent-foo'
      at Interceptor.body (tests/index.js:900:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  9) Thorn Agent request methods should send PUT request:
     Uncaught AssertionError: expected undefined to equal 'Agent-foo'
      at Interceptor.body (tests/index.js:920:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  10) Thorn Agent request methods should send DELETE request:
     Uncaught AssertionError: expected undefined to equal 'Agent-foo'
      at Interceptor.body (tests/index.js:940:64)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  11) Utils login should post to the login endpoint with the right params:
     Uncaught AssertionError: expected undefined to equal 'TestHeader'
      at Interceptor.body (tests/utils.js:61:60)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)

  12) Utils refresh should post to the refresh endpoint with the right params:
     Uncaught AssertionError: expected undefined to equal 'TestHeader'
      at Interceptor.body (tests/utils.js:102:60)
      at end (node_modules/nock/lib/request_overrider.js:321:34)
      at OverriddenClientRequest.RequestOverrider.req.end (node_modules/nock/lib/request_overrider.js:160:7)
      at Request.end (node_modules/request/request.js:1438:14)
      at end (node_modules/request/request.js:553:16)
      at Immediate.<anonymous> (node_modules/request/request.js:580:7)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,6 +10,3 @@ script:
 
 after_script: "cat /tmp/coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
 
-matrix:
-  allow_failures:
-  - node_js: "8"

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "jsdoc-babel": "^0.3.0",
     "mocha": "^3.2.0",
     "mocha-junit-reporter": "^1.13.0",
-    "nock": "^9.0.7",
+    "nock": "^9.0.14",
     "sinon": "^2.1.0"
   },
   "dependencies": {

